File: Sakila_Net/VB_SakilaLib/Neg/NegCity.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegCity
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOCity.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( city as ClCity) as Boolean
                        Return DAO.DAOCity.SelectOne(city)     
                    End Function
                    Public  Shared Function crear( city as ClCity) as Integer
                        Return DAO.DAOCity.SqlInsert(city)     
                    End Function
                    Public  Shared Function eliminar( city as ClCity) as Integer
                        Return DAO.DAOCity.SqlDelete(city)     
                    End Function
                    Public  Shared Function actualizar( city as ClCity) as Integer
                        Return DAO.DAOCity.SqlUpdate(city)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/CS_SakilaLib/Neg/NegCity.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegCity
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOCity.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClCity city) 
        {
            return DAO.DAOCity.sqlSelect(city);
        }
        public  static int crear(ClCity city)
        {
            return DAO.DAOCity.sqlInsert(city);
        }
        public  static int eliminar(ClCity city)
        {
            return DAO.DAOCity.sqlDelete(city);
        }
        public  static int actualizar(ClCity city)
        {
            return DAO.DAOCity.sqlUpdate(city);
        }
    }
}
